atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/KvsSweepQueueTest.java

```diff
@@ -241,4 +397,21 @@ private void assertReadAtTimestampReturnsNothing(TableReference tableRef, long r
         Map<Cell, Long> singleRead = ImmutableMap.of(DEFAULT_CELL, ts);
         return kvs.get(tableRef, singleRead);
     }
+
+    private void assertProgressUpdatedToFineTimestampPartition(long partitionFine) {
+        assertThat(sweepQueue.tables.progress.getLastSweptTimestampPartition(ShardAndStrategy.conservative(CONS_SHARD)))
```

Comment:
Demeter is displeased with you for doubly breaking her Law :zap:

Reply:
Doubly visible for testing? I guess I can have a new KvsSweepProgress to check it out 🦌

Reply:
I meant that you are reaching into the internals of sweepQueue, and then into the internals of tables, to access functionality exposed by progress. Merging a couple of the objects will help, as will allowing accessors. Doing it only in test methods is less bad then also doing it in production code.